services/resource-coordinator/src/dao/FlowsK8sDao.js

```diff
@@ -0,0 +1,42 @@
+const Lib = require('backendCommonsLib');
+const { Flow } = Lib;
+const { FlowsDao } = require('@openintegrationhub/resource-coordinator');
+
+class FlowsK8sDao extends FlowsDao {
+    constructor(k8s) {
+        super();
+        this._crdClient = k8s.getCRDClient();
+    }
+
+    async findById(id) {
+        return new Flow((await this._crdClient.flows(id).get()).body);
+    }
+
+    update(flow) {
+        return this._crdClient.flow(flow.id).put({
+            body: flow.toCRD()
+        });
+    }
+
+    async findAll() {
+        return ((await this._crdClient.flows.get()).body.items || []).map(item => new Flow(item));
+    }
+
+    async ensureFinalizer(flow) {
```

Comment:
Just to think about:
maybe we may implement flow model as active record?
1) this makes  interface of flows dao smaller (only CRUD methods)  and consequently more generic
2) At least in my opinion: ensureFinializer and removeFinalizer are property of Flow itself, not DAO layer.
3) it's clear how to implement it: flowsDao is actually fabric that creates flows, so it may inject reference to itself into Flow. And then flow may implement several methods like save/update/delete on itself, that proxies changes to dao